mirte_telemetrix/scripts/ROS_telemetrix_aio_api.py

```diff
@@ -494,19 +532,19 @@ async def set_speed(self, speed):
         if self.prev_motor_speed != speed:
             await self.init_motors()
             if speed >= 0:
-                await self.board.digital_write(self.pins["in1"], 0)
-                await self.board.digital_write(self.pins["in2"], 1)
+                await self.board.digital_write(self.pins["d1"], 0)
+                await self.board.digital_write(self.pins["d2"], 1)
                 await analog_write(
                     self.board,
-                    self.pins["en"],
+                    self.pins["p1"],
                     int(min(speed, 100) / 100.0 * board_mapping.get_max_pwm_value()),
                 )
             elif speed < 0:
-                await self.board.digital_write(self.pins["in2"], 0)
-                await self.board.digital_write(self.pins["in1"], 1)
+                await self.board.digital_write(self.pins["d2"], 0)
+                await self.board.digital_write(self.pins["d1"], 1)
```

Comment:
wouldn't it be nice to first set all digital pins to 0, set the pwm value and then set one of the digital pins?

Reply:
Yes, done!